lava_test_plans/projects/lkft-android/include/metadata.jinja2

```diff
@@ -0,0 +1,22 @@
+  android.build: "{{ BUILD_NUMBER }}"
+  android.name: "{{ JOB_NAME }}"
+  android.url: "{{ REFERENCE_BUILD_URL }}"
+  android.version: "{{ ANDROID_VERSION }}"
+{% if ANDROID_VENDOR_FINGERPRINT is defined %}
+  android.build.vendor.fingerprint: "{{ ANDROID_VENDOR_FINGERPRINT }}"
+{% endif %}
+{% if ANDROID_GSI_FINGERPRINT is defined %}
+  android.build.gsi.url: "{{ ANDROID_GSI_URL }}"
+  android.build.gsi.fingerprint: "{{ ANDROID_GSI_FINGERPRINT }}"
+{% endif %}
+{% if TUXSUITE_DOWNLOAD_URL is defined %}
+  tuxsuite.download.url: "{{ TUXSUITE_DOWNLOAD_URL }}"
+{% endif %}
+  lkft.build.config: "{{ LKFT_BUILD_CONFIG }}"
+  git branch: "{{ KERNEL_BRANCH }}"
+  git repo: "{{ KERNEL_REPO }}"
+  git commit: "{{ KERNEL_COMMIT }}"
+  git describe: "{{ KERNEL_DESCRIBE }}"
+  build-url: "{{ BUILD_URL }}"
+  toolchain: "{{ TEST_METADATA_TOOLCHAIN }}"
```

Comment:
isn't prefix 'TEST_METADATA_' unnecessary ?

Reply:
TEST_METADATA means it's only used as TEST_METADATA, not build metadata, etc

Reply:
What do you mean with TEST_METADATA?

Reply:
TEST means the variable is on lava test job, TEST_METADATA_ means the variable is used for the lava test job metadata section.
But here on this toolchain metatdata, as long as I know, it is not that useful for the Android failures investigation, and with the tuxsuite bake command it's impossible to get the toolchain information now, so I am thinking if it should be removed. what do you think about it?